docs/dev/examples.md

```diff
@@ -0,0 +1,28 @@
+# Examples
+
+This document gives an overview of example structure.
+
+## Concrete Python
+
+### How to create an example?
+
+There are two options to choose from:
+
+- Option A)
+  - Create `examples/foo/foo.ipynb` and `examples/foo/foo_utils.py`
```

Comment:
Could you update a bit:
- remove 

> and examples/foo/foo_utils.py
Write a tutorial in the notebook
Copy the code to utils
Add tests for the implementation in tests/execution/test_examples.py

- say it will be tested by the `make test-notebooks`

Reply:
Sure .